From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

Tests can fail due to ambiguous module. Use PackageImports extension as workaround

    examples/Both.hs:10:1: error:    
        Ambiguous module name ‘Data.Kind’:    
          it was found in multiple packages: base-4.15.1.0 kinds-0.0.1.5

diff -urN record-dot-preprocessor-0.2.14/examples/Both.hs record-dot-preprocessor-0.2.14-r1/examples/Both.hs
--- record-dot-preprocessor-0.2.14/examples/Both.hs	2001-09-08 19:46:40.000000000 -0600
+++ record-dot-preprocessor-0.2.14-r1/examples/Both.hs	2022-02-22 21:07:13.767317606 -0700
@@ -1,7 +1,7 @@
 -- Test for everything that is supported by both the plugin and the preprocessor
 
 {-# OPTIONS_GHC -Werror -Wall -Wno-type-defaults -Wno-partial-type-signatures -Wincomplete-record-updates -Wno-unused-top-binds #-} -- can we produce -Wall clean code
-{-# LANGUAGE PartialTypeSignatures, GADTs, StandaloneDeriving, DataKinds, KindSignatures #-} -- also tests we put language extensions before imports
+{-# LANGUAGE PartialTypeSignatures, GADTs, StandaloneDeriving, DataKinds, KindSignatures, PackageImports #-} -- also tests we put language extensions before imports
 -- 8.8+ doesn't need it to be set explicitly
 {-# LANGUAGE ExistentialQuantification #-}
 
@@ -9,7 +9,7 @@
 import Data.Version
 import Data.Proxy
 import Data.Functor.Identity (Identity(..))
-import qualified Data.Kind as T
+import qualified "base" Data.Kind as T
 
 
 main :: IO ()
diff -urN record-dot-preprocessor-0.2.14/test/PluginExample.hs record-dot-preprocessor-0.2.14-r1/test/PluginExample.hs
--- record-dot-preprocessor-0.2.14/test/PluginExample.hs	2001-09-08 19:46:40.000000000 -0600
+++ record-dot-preprocessor-0.2.14-r1/test/PluginExample.hs	2022-02-22 21:09:39.447317677 -0700
@@ -1,4 +1,5 @@
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE PackageImports #-}
 
 #if __GLASGOW_HASKELL__ < 806
 
